filter/etc/e-smith/templates/etc/rspamd/whitelist_IP.map/10base

```diff
@@ -0,0 +1,16 @@
+
+#
+# whitelist IP in rspamd
+#
+
+{
+if(defined $postfix{AccessBypassList}) {
+    foreach my $ip (split(',', ($postfix{AccessBypassList}))) {
+    push @whitelist, $ip;
+    }
+
+    foreach my $lines (@whitelist){
+    $OUT .= "$lines\n";
+    }
+}
+}
```

Comment:
Please fix the block indentation. I can't see if there are mixed tab/spaces.

Proposal: this block could be implemented in a much shorter way

```text
{
    $OUT = $postfix{'AccessBypassList'};
    $OUT =~ s/,/\n/g;
}
```